src/api/http.rs

```diff
@@ -234,7 +235,7 @@ fn change_partition_state(
     let task = async move {
         match action {
             Action::Attach => group.attach(partition_id).await,
-            Action::Detach => group.detach(partition_id).await,
+            Action::Detach => group.detach(partition_id).await.map(|_| ()),
```

Comment:
Why do you need to map to zero type?

Reply:
I get it, maybe just:
```
{
    group.detach(partition_id).await;
}
```
?

Reply:
Oh, sure, missed that opportunity :)

Reply:
Oh, wait, actually we need error from that result, and map applies to Ok only